### PR TITLE
marchack approval message added

### PR DESCRIPTION
**Pull request by TheMarchack**
added marchack page and changed index.html with a link to marchack.html

### CODE REVIEW

**Review by VernersGavilovskis — CHANGES_REQUESTED** [review]
🥒

File: index.html [hunk 1]
```diff
@@ -45,6 +45,7 @@ <h4 class="text-white">Contact</h4>
             <a href="#" class="navbar-brand d-flex align-items-center">
                 <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="mr-2"><path d="M23 19a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h4l2-3h6l2 3h4a2 2 0 0 1 2 2z"></path><circle cx="12" cy="13" r="4"></circle></svg>
                 <strong>Album</strong>
+                <a href="marchack.html">Approval</a>
```
Comment (VernersGavilovskis) [inline index.html]: ŠIs neder 🥒

**Review by VernersGavilovskis — CHANGES_REQUESTED** [review]
🌵

File: index.html (hunk 1 above)
Comment (TheMarchack) [reply index.html]: gonna change it ;)

**Review by VernersGavilovskis — APPROVED** [review]
Viss labi

**Pull request merged**